lib/rules/prop-name-casing.js

```diff
@@ -71,6 +71,7 @@ function create (context) {
 
 module.exports = {
   meta: {
+    type: 'layout',
```

Comment:
I feel this is `suggestion` because this is about neither whitespaces nor punctuators.